gossip/src/restart_crds_values.rs

```diff
@@ -29,6 +29,25 @@ pub enum RestartLastVotedForkSlotsError {
     LastVotedForkEmpty,
 }
 
+#[derive(Serialize, Deserialize, Clone, PartialEq, Eq, AbiExample, Debug)]
+pub struct RestartHeaviestFork {
+    pub from: Pubkey,
+    pub wallclock: u64,
+    pub last_slot: Slot,
+    pub last_slot_hash: Hash,
+    // Sum of received heaviest fork validator stake / total stake * u16::MAX.
+    received_heaviest_fork_ratio: u16,
```

Comment:
I would probably call this `stake_ratio` or `observed_stake_ratio`.
`heaviest_fork` is already in the name of struct; it is redundant to repeat it here again.

Reply:
also why this `u16` better than sending a `f64` or the `(stake, total_stake)` as a tuple?

Reply:
The original idea is we can save some bytes, but now you mention it, it's not really worth losing the precision by switching from 16 bytes to 2 bytes, I'm changing to (stake, total_stake) now.